test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -1856,16 +1856,16 @@ TEST_P(ExtProcIntegrationTest, RequestMessageNewTimeoutNoMutation) {
 TEST_P(ExtProcIntegrationTest, RequestMessageNoMutationMultipleNewTimeout) {
   // Set envoy filter timeout to be 200ms.
   proto_config_.mutable_message_timeout()->set_nanos(200000000);
-  // Config max_message_timeout proto to 10s to enable the new timeout API.
-  proto_config_.mutable_max_message_timeout()->set_seconds(10);
+  // Config max_message_timeout proto to 60s to enable the new timeout API.
+  proto_config_.mutable_max_message_timeout()->set_seconds(60);
   initializeConfig();
   HttpIntegrationTest::initialize();
   auto response = sendDownstreamRequest(absl::nullopt);
 
   processRequestHeadersMessage(*grpc_upstreams_[0], true,
                                [this](const HttpHeaders&, HeadersResponse&) {
-                                 // Send a 500ms new timeout update first.
-                                 serverSendNewTimeout(500);
+                                 // Send a 60s new timeout update first.
+                                 serverSendNewTimeout(60000);
```

Comment:
20x increase here, where is the time going?

Reply:
Changed this to TestUtility::DefaultTimeout() which is 10s in non ASAN/TSAN cases.  The idea is to avoid flaky tests due to the response doesn't come back within in that 500ms due to busy CPU or other reasons.

Reply:
Setting this timer to 500ms, then have the ext_proc service to wait for 300ms is a problematic logic which relies on the ext_proc service can handle the request and send back the response within 200ms, which maybe okay in normal scenarios, but may not work in TSAN/ASAN with repeatedly tests, and leads to flaky.